sensing/autoware_pointcloud_preprocessor/schema/cocatenate_and_time_sync_node.schema.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Concatenate and Time Synchronize Node",
+  "type": "object",
+  "definitions": {
+    "concatenate_and_time_sync_node": {
+      "type": "object",
+      "properties": {
+        "maximum_queue_size": {
+          "type": "integer",
+          "default": 5,
```

Comment:
@mojomex 
The `maximum_queue_size` is defined for most of the nodes in pointcloud preprocessor (and the value is 5). 
I think if we modify the number from 5 to 2, perhaps we also need to consider whether we should modify the value in other nodes. 

btw,  `maximum_queue_size` parameter is for the qos setting instead of the group (collector?)

Reply:
As I just discussed with @drwnz in person, setting it to `1` is best. This reduces latency to a minimum.

Reply:
From the bench experiment, setting to 1 will reduce the frequency if the network is congested (which makes sense).
Yesterday, when I adjusted it to 1, the hz looked similar (see [INTERNAL_LINK](https://tier4.atlassian.net/wiki/x/LwNhyQ)).

However, I would like to adjust this parameter to 1 in other PR (with other nodes together), because I think this needs more investigation :)